dunwoody_disaster/views/MapScreen.py

```diff
@@ -136,16 +139,17 @@ def from_json(json: dict, char: Character) -> "MapScreen":
         return map
 
     @staticmethod
-    def build_map(char: Character) -> "MapScreen":
-        test_enemy = CharacterFactory.createTestChar()
-        test_enemy.name = "test enemy"
+    def defaultMap(char: Character) -> "MapScreen":
         ms = MapScreen(char, None)
+        chars = CharacterFactory
+
         ms.setAsset("MainMap")
-        ms.addRoom("Bus Stop", (419, 700), test_enemy, "no_texture")
-        ms.addRoom("Court Yard", (693, 559), test_enemy, "no_texture")
-        ms.addRoom("Commons", (451, 449), test_enemy, "no_texture")
-        ms.addRoom("Math", (236, 359), test_enemy, "no_texture")
-        ms.addRoom("English", (770, 366), test_enemy, "no_texture")
-        ms.addRoom("Science", (490, 217), test_enemy, "no_texture")
-        ms.addRoom("Dean's Office", (90, 589), test_enemy, "no_texture")
+        ms.addRoom("Bus Stop", (419, 700), chars.JoeAxberg(), "no_texture")
+        ms.addRoom("Court Yard", (693, 559), chars.LeAnnSimonson(), "no_texture")
+        ms.addRoom("Commons", (451, 449), chars.RyanRengo(), "no_texture")
+        ms.addRoom("Math", (236, 359), chars.NoureenSajid(), "no_texture")
+        ms.addRoom("English", (770, 366), chars.enemy1(), "no_texture")
+        ms.addRoom("Science", (490, 217), chars.enemy2(), "no_texture")
+        ms.addRoom("Dean's Office", (90, 589), chars.BillHudson(), "no_texture")
```

Comment:
@EngineeringDude123 would you be able to source images for these? I propose we use 1024x1024 character images. I can help with resizing if it's a bit off, but getting as close as possible to the target resolution is going to have the least impact on quality.

@EngineeringDude123 @MLHumphries @NoahBartell I need two other teachers to put in, any suggestions?

Reply:
@chanson02 You could add Rob Bentz and Amalan Pulendran

Reply:
I never had classes with Rob or John, I'd be okay adding Amalan and Matthew though.

Reply:
Bill Hudson

Reply:
There's a teacher named David White who is basically a Ryan Rengo for the Cloud engineering students. We can add him as a DLC boss for my buddies in that program